<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<!-- args to be passed in -->
	<arg name="videopath" default="" />
	<arg name="bagpath" default="" />
	<arg name="imgtopic" default="/image_raw" />
	<arg name="show_tracks_video" default="false" />

	<include file="$(find visual_mtt)/launch/play_from_recording.launch">
		<arg name="videopath" value="$(arg videopath)" />
		<arg name="bagpath" value="$(arg bagpath)" />
		<arg name="imgtopic" value="$(arg imgtopic)" />
		<arg name="show_tracks_video" value="$(arg show_tracks_video)" />
	</include>

	<node pkg="web_video_server" type="web_video_server" name="web_video_server"/>
	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
	<node name="publish_topics_on_network" pkg="visual_mtt" type="publish_topics_on_network.sh"/>
</launch>